profiles/common/modules/features/multisite_custom_error/multisite_custom_error.install

```diff
@@ -11,25 +11,27 @@ include_once 'multisite_custom_error.install.inc';
  * Implements hook_enable().
  */
 function multisite_custom_error_enable() {
-  //Activation message
+  // Activation message.
   drupal_set_message(t('Multisite Custom Error %v feature is now active on your site.', array('%v' => _multisite_custom_error_get_version())));
 
-  // configuration for the system settings and the customerror module
+  // Configuration for the system settings and the customerror module.
   module_load_include('module', 'customerror');
   foreach (_customerror_enum_errors() as $code => $desc) {
     variable_set('site_' . $code, 'customerror/' . $code);
     variable_set('customerror_' . $code . '_theme', 'ec_resp');
     drupal_set_message(t('Default %v error page has been set.', array('%v' => $code)));
   }
-  drupal_set_message(t('Go to ' . l('admin/config/system/site-information', 'admin/config/system/site-information', array('attributes' => array('target' => '_blank'))) . ' to review the settings.'));
-  drupal_set_message(t('Go to ' . l('admin/config/system/customerror', 'admin/config/system/customerror', array('attributes' => array('target' => '_blank'))) . ' to review additional configuration options.'));
+  $link = l(t('admin/config/system/site-information'), 'admin/config/system/site-information', array('attributes' => array('target' => '_blank')));
```

Comment:
It's not a good idea to expose 'admin/config/system/site-information' as a translatable string. The people from the translation teams wouldn't know what to do with it. Instead of the path, just use the page title: `t('Site information')`.